completions/.gitignore

```diff
@@ -0,0 +1,5 @@
+CMakeCache.txt
```

Comment:
Delete this .gitignore file. (See my blocking comment in completions/README.md.)

Reply:
`File deleted`